smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -202,9 +203,15 @@ GSC.ConnectorApp.Window.DevicesDisplaying.initialize = function() {
       (GSC.ObjectHelpers.extractKey(
           GSC.PopupWindow.Client.getData(), 'readerTrackerUnsubscriber'));
   // Stop tracking the current list of readers when our window gets closed.
-  chrome.app.window.current().onClosed.addListener(function() {
-    readerTrackerUnsubscriber(onReadersChanged);
-  });
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP) {
+    chrome.app.window.current().onClosed.addListener(function() {
+      readerTrackerUnsubscriber(onReadersChanged);
+    });
+  } else if (GSC.Packaging.MODE == GSC.Packaging.Mode.EXTENSION) {
+    chrome.windows.onRemoved.addListener(function() {
```

Comment:
The documentation of chrome.windows.onRemoved is a bit unclear... Is it actually triggered when our window (which is the browser action popup) closed?